src/Tizen.Security.WebAuthn/Tizen.Security.WebAuthn/AuthenticationExt.cs

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using static Interop;
+using static Tizen.Security.WebAuthn.ErrorFactory;
+
+namespace Tizen.Security.WebAuthn
+{
+    /// <summary>
+    /// Authenticator extension
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public class AuthenticationExt
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AuthenticationExt"/> class.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        /// <param name="extensionId">
+        /// Extension Identifier defined in the following registry.
+        /// https://www.iana.org/assignments/webauthn/webauthn.xhtml#webauthn-extension-ids
```

Comment:
It would be best to use links in the text like this:
```suggestion
        /// Extension Identifier defined in the <a href="https://www.iana.org/assignments/webauthn/webauthn.xhtml#webauthn-extension-ids">WebAuthn Extension Identifiers registry</a>.
```
but for now lets just replace period with a colon ("registry. https..." -> "registry: https...")

Reply:
Done